src/main/java/com/post/menagment/model/PostOffice.java

```diff
@@ -0,0 +1,42 @@
+package com.post.menagment.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class PostOffice {
+    @Id
+    @GeneratedValue
+    private long id;
```

Comment:
It's recommended to declare consistently-named identifier properties on persistent classes and that you use a nullable (i.e., non-primitive) type. So, please, use wrapper type.

http://docs.jboss.org/hibernate/core/3.3/reference/en/html/persistent-classes.html#persistent-classes-pojo-identifier